### ISSUE TITLE
Modules are too long

### ISSUE THREAD

**Issue by PoorRican** [PoorRican/ncbi-rs#8]
In an attempt to conform to orig ASN.1 spec, modules where kept at the same exact length. This has produced modules that are hard to read and understand. In the original spec, certain definitions are broken into sections. This should be emulated in this implementation. Particularly `seq` and `seqfeat`

**Comment by smoe** [comment]
I disagree - just a bit - the design flaw is to manually create all those implementations in the first place. Instead, everything should be autogenerated just with the ASN1 files as input. Once that is established, nobody will look at the generated files in the first place - errors would only be addressed in the generator.
